app/code/community/Reverb/ReverbSync/sql/reverbsync_setup/mysql4-upgrade-0.1.12-0.1.13.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * Reverb
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
```

Comment:
Can we please exclude licensing info from the files? The entire project will be licensed under Apache 2.0